.github/workflows/compile_examples.yml

```diff
@@ -0,0 +1,25 @@
+name: Compile Examples
+
+on: [pull_request]
+
+jobs:
+  unittests-linux-generic:
```

Comment:
Fail 😆

![image](https://user-images.githubusercontent.com/2820734/149747036-74981769-5759-450a-abab-61e1f8ff8fc3.png)

Reply:
I also missed that you requested a review from @chris-durand and just merged it… oops

Reply:
Force push?

Reply:
Nah, the CI job name is not that important to justify a force push.